libmscore/stafftype.cpp

```diff
@@ -1,4 +1,4 @@
-//=============================================================================
+
```

Comment:
This was probably a typo - sometimes my system (my laptop is about 7 years old) is hanging, or it seems at least. Probably the cursor had been in the wrong place ...